docs/DesReqGui/source/core_kpi.rst

```diff
@@ -51,6 +51,57 @@ Energy use in a given period of time
    any type; finally, :math:`P_i` is the instantaneous power used by the
    energy vector :math:`i`; :math:`A` is the total floor area of the building.
 
+Peak electricity demand in a given period of time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   This KPI is the measure of the site HVAC peak electricity use in :math:`kW/m^2`
+   when accounting for the sum of all electrical equipment present in the test
+   case HVAC system.
+   The mathematical formulation for this KPI is the following:
+
+   .. math:: P_E(t_0, t_f) = \frac{\max_{15}[\sum_{i\in I} P_i(t)]}{A}
+
+   Where :math:`P_E(t_0, t_f)` is the peak electricity demand from the
+   initial time :math:`t_0` up to the final time :math:`t_f`; :math:`I` denotes
+   the set of equipment in the system with an associated energy use of
+   electricity; :math:`P_i` is the instantaneous power used by the
+   equipment :math:`i`; :math:`A` is the total floor area of the building; and
+   :math:`max_{15}[]` indicates the maximum value after taking the average of each 15 minute interval.
+
+Peak gas demand in a given period of time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   This KPI is the measure of the site HVAC peak gas use in :math:`kW/m^2`
+   when accounting for the sum of all gas equipment present in the test
+   case HVAC system.
+   The mathematical formulation for this KPI is the following:
+
+   .. math:: P_G(t_0, t_f) = \frac{\max_{15}[\sum_{i\in I} P_i(t)]}{A}
+
+   Where :math:`P_G(t_0, t_f)` is the peak gas demand from the
+   initial time :math:`t_0` up to the final time :math:`t_f`; :math:`I` denotes
+   the set of equipment in the system with an associated energy use of
+   gas; :math:`P_i` is the instantaneous power used by the
+   equipment :math:`i`; :math:`A` is the total floor area of the building; and
+   :math:`max_{15}[]` indicates the maximum value after taking the average of each 15 minute interval.
+
+Peak district heating demand in a given period of time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   This KPI is the measure of the site HVAC peak district heating use in :math:`kW/m^2`
+   when accounting for the sum of all district heating equipment present in the test
+   case HVAC system.
+   The mathematical formulation for this KPI is the following:
+
+   .. math:: P_{DH}(t_0, t_f) = \frac{\max_{15}[\sum_{i\in I} P_i(t)]}{A}
+
+   Where :math:`P_{Dh}(t_0, t_f)` is the peak district heating demand from the
```

Comment:
```suggestion
   Where :math:`P_{DH}(t_0, t_f)` is the peak district heating demand from the
```
Sorry I didn't notice the capitalization in my previous suggestion. Just to be coherent with the symbols used in the equation above.